roles/edpm_nova/templates/02-nova-host-specific.conf.j2

```diff
@@ -1,2 +1,9 @@
 [DEFAULT]
 my_ip = {{ ctlplane_ip }}
+[os_vif_ovs]
+ovsdb_connection="unix:/var/run/openvswitch/db.sock"
+{% if edpm_nova_tls_certs_enabled | bool %}
+[libvirt]
+live_migration_with_native_tls = true
+live_migration_scheme = tls
+{% endif %}
```

Comment:
OK, we agreed that we don't have information in nova-operator if TLS is enabled in the dataplane or not, so we cannot provide this information via the cell config. If in the future we make every nova compute config a j2 template then we can  move this template logic to the cell config. However we might not want to as that will couple the nova-operator with the edpm ansible vars.